test/elixir_search_extractor/search_keyword/search_keywords_test.exs

```diff
@@ -48,6 +48,24 @@ defmodule ElixirSearchExtractor.SearchKeyword.SearchKeywordsTest do
     end
   end
 
+  describe "get_user_keyword/2" do
```

Comment:
Let's add another test case when getting a keyword that belongs to the other user.